docs/pipeline.rst

```diff
@@ -167,6 +193,78 @@ intent_classifier_sklearn
           # This is used with the ``C`` hyperparameter in GridSearchCV.
           kernels: ["linear"]
 
+intent_classifier_tensorflow_embedding
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:Short: Embedding intent classifier
+:Outputs: ``intent`` and ``intent_ranking``
+:Output-Example:
+
+    .. code-block:: json
+
+        {
+            "intent": {"name": "greet", "confidence": 0.8343},
+            "intent_ranking": [
+                {
+                    "confidence": 0.385910906220309,
+                    "name": "goodbye"
+                },
+                {
+                    "confidence": 0.28161531595656784,
+                    "name": "restaurant_search"
+                }
+            ]
+        }
+
+:Description:
+    The embedding intent classifier embeds user inputs and intent labels into the same space. Supervised embeddings are
+    trained by maximizing similarity between them. This algorithm is based on
+    the starspace idea from: https://arxiv.org/abs/1709.03856.
```

Comment:
mention that by default we have some hidden layers and dropout so it's not quite starspace